Source/WebCore/PAL/pal/avfoundation/OutputContext.mm

```diff
@@ -77,16 +75,11 @@
         return [m_context deviceName];
 
     StringBuilder builder;
-    auto devices = outputDevices();
-    auto iterator = devices.begin();
-
-    while (iterator != devices.end()) {
-        builder.append(iterator->name());
-
-        if (++iterator != devices.end())
+    for (auto& device : outputDevices()) {
+        if (!builder.isEmpty())
             builder.append(" + "_s);
+        builder.append(device.name());
     }
-
     return builder.toString();
```

Comment:
@weinig added something to `StringConcatenate.h` to do this with a lambda that I think is nicer. I can't remember the syntax offhand.

Reply:
Probably `interleave()`